auth/src/main/java/org/aerogear/mobile/security/checks/RootedCheck.java

```diff
@@ -19,18 +15,20 @@
     /**
      * Check whether the device is rooted or not.
      *
-     * @param context Context to be used by the check.
-     * @return <code>true</code> if the device is rooted.
+     * @param context {@link Context} to be used by the check
+     * @return <code>true</code> if the device is rooted
+     * @throws IllegalArgumentException if context is null
```

Comment:
IllegalArgumentException is not thrown here. Context is guaranteed to be non null by the parent class.